webapp/src/components/Registration/Signup/signup.js

```diff
@@ -24,10 +24,12 @@ class SignUp extends Component {
         error: 'Passwords do not match!',
       });
     }
-    fetch(`http://localhost:8888/registration`, {
+    fetch('http://localhost:8888/registration', {
```

Comment:
I don't think this will work as https right now, but it will be necessary for it to be https in production otherwise the email/password will be sent in the clear.

Reply:
How should we handle this? Should I just use https right now and it will default to not https? Or should we have a better way of handling URL's for our API.

Reply:
👍

Reply:
For me, it hangs up trying to the endpoint. It never reaches the Tornado server, I think.